graphql/schema.graphql

```diff
@@ -765,11 +767,13 @@ type Task {
   spawnHostLink: String
   startTime: Time
   status: String!
+  originalStatus: String
   taskGroup: String
   taskGroupMaxHosts: Int
   timeTaken: Duration
   totalTestCount: Int!
-  version: String! @deprecated(reason: "version is deprecated. Use versionMetadata instead.")
+  version: String!
```

Comment:
I think the directive has to be placed on the same line as the field. https://www.apollographql.com/docs/apollo-server/schema/directives/

Reply:
oof, my linter does it automatically on save.